vtex-sites/starter.store
# feat: Generates CMS webhook URLs json file from the store's config

## What's the purpose of this pull request?

Uses the new `vtexHeadlessCms` config.

## How to test it?

- Run `yarn dev` or `yarn build` and check the hidden `.faststore` folder, the file `cms-webhook-urls.json` should be there.

### Faststore related PRs

https://github.com/vtex/faststore/pull/1829
